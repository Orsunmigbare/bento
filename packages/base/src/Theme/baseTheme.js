import { ThemeManager } from "@staccx/theme"

export const theme = {
  name: "Default theme",
  spacing: {
    huge: "96px",
    large: "48px",
    mediumPlus: "32px",
    medium: "24px",
    small: "12px",
    tiny: "6px",
    micro: "3px"
  },
  wrapper: {
    small: "420px",
    medium: "640px",
    large: "1024px",
    breakout: "640px"
  },
  color: {
    white: "#fff",
    bg: "#fff",
    subtleHover: "#F7F7FF",
    bgGray: "#f7f7ff",
    line: "#EBEBF2",
    gray: "#767676",
    wcag: "#777",
    disabled: "#BABABA",
    primary: "#00a190",
    secondary: "#F5A623",
    text: "#2b2b2b",
    negative: "#ea526f",
    positive: "#4A90E2",
    warning: "#c54059",
    black: "#000",
    green: "#00a190" // for toggle eks
  },
  fontFamily: {
    heading: "sans-serif",
    body: "sans-serif"
  },
  font: {
    h1: "36px",
    h2: "26px",
    h3: "22px",
    h4: "18px",
    h5: "16px",
    h6: "14px",
    base: "14px",
    tiny: "12px",
    input: "16px",
    huge: "44px"
  },
  targetSize: {
    small: "36px",
    normal: "44px"
  },
  borderRadius: "2px"
}

const manager = new ThemeManager(theme)
export default manager
