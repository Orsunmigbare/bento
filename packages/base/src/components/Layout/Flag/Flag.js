import React from "react"
import PropTypes from "prop-types"
import styled, { css } from "styled-components"

const Flag = ({
  children,
  className,
  img,
  reverse,
  top,
  bottom,
  responsive,
  flush,
  large,
  ...otherProps
}) => (
  <FlagObject
    className={className}
    reverse={reverse}
    top={top}
    bottom={bottom}
    responsive={responsive}
    flush={flush}
    large={large}
    {...otherProps}
  >
    <FlagImg>{img}</FlagImg>
    <FlagBody>{children}</FlagBody>
  </FlagObject>
)

const FlagImg = styled.div`
  display: table-cell;
  width: 1px;
  vertical-align: middle;
  padding-right: ${p => p.theme.spacing.medium};
  > img {
    max-width: none;
  }
`

const FlagBody = styled.div`
  display: table-cell;
  vertical-align: middle;
  width: auto;

  &,
  > :last-child {
    margin-bottom: 0;
  }
`

const reverse = css`
  direction: rtl;
  > ${FlagImg}, > ${FlagBody} {
    direction: ltr;
  }

  > ${FlagImg} {
    padding-right: 0;
    padding-left: ${p => p.theme.spacing.medium};
  }
`

const reverseLarge = css`
  > ${FlagImg} {
    padding-right: 0;
    padding-left: ${p => p.theme.spacing.large};
  }
`

const top = css`
  > ${FlagImg}, > ${FlagBody} {
    vertical-align: top;
  }
`

const bottom = css`
  > ${FlagImg}, > ${FlagBody} {
    vertical-align: bottom;
  }
`

const flush = css`
  > ${FlagImg} {
    padding-right: 0;
    padding-left: 0;
  }
`

const large = css`
  > ${FlagImg} {
    padding-right: ${p => p.theme.spacing.large};
  }
  ${props => (props.reverse ? reverseLarge : null)};
`

const responsive = css`
  @media screen and (max-width: ${props => props.responsive}px) {
    &,
    > ${FlagImg}, > ${FlagBody} {
      display: block;
      direction: initial;
      padding: 0;
    }
    > ${FlagImg} {
      width: auto;
      padding-bottom: ${p => p.theme.spacing.medium};
    }

    ${FlagImg} > img {
      max-width: 100%;
    }
  }
`

const FlagObject = styled.div`
  display: table;
  width: 100%;
  border-spacing: 0;
  ${props => (props.reverse ? reverse : null)};
  ${props => (props.top ? top : null)};
  ${props => (props.bottom ? bottom : null)};
  ${props => (props.flush ? flush : null)};
  ${props => (props.large ? large : null)};
  ${props => (props.responsive > 0 ? responsive : null)};
`

Flag.defaultProps = {
  className: "",
  reverse: null,
  top: null,
  bottom: null,
  responsive: 0,
  flush: null,
  large: null
}

Flag.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.element),
    PropTypes.element,
    PropTypes.string,
    PropTypes.array
  ]).isRequired,
  img: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.element),
    PropTypes.element,
    PropTypes.string,
    PropTypes.array
  ]).isRequired,
  className: PropTypes.string,
  reverse: PropTypes.bool,
  top: PropTypes.bool,
  bottom: PropTypes.bool,
  responsive: PropTypes.number,
  flush: PropTypes.bool,
  large: PropTypes.bool
}

export default Flag
