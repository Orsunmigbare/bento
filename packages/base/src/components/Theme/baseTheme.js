export const theme = {
  name: "StaccXBase",
  spacing: {
    huge: "96px",
    large: "48px",
    mediumPlus: "32px",
    medium: "24px",
    small: "12px",
    tiny: "6px"
  },
  wrapper: {
    small: "420px",
    medium: "640px",
    large: "1024px"
  },
  color: {
    bg: "#fff",
    bgGray: "#f7f7ff",
    primary: "#00a190",
    gray: "#767676",
    text: "#2b2b2b",
    white: "#fff",
    red: "#ea526f",
    green: "#00a190",
    blue: "#988EFB",
    yellow: "#f4da87",
    line: "#EBEBF2",
    wcag: "#777",
    disabled: "#BABABA",
    warning: "#c54059",
    black: "#000",
    subtleHover: "#F7F7FF"
  },
  font: {
    size: {
      h1: "36px",
      h2: "26px",
      h3: "22px",
      h4: "18px",
      h5: "16px",
      h6: "14px",
      base: "14px",
      tiny: "12px",
      input: "16px",
      huge: "44px"
    },
    heading: "Libre Baskerville",
    body: "Libre Franklin"
  },
  targetSize: {
    small: "36px",
    normal: "44px"
  },
  borderRadius: "2px"
}

export default theme
