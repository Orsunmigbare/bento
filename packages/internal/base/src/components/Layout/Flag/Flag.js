import React from "react"
import PropTypes from "prop-types"
import styled, { css } from "styled-components"
import { spacing, themify, themeProps } from "@staccx/theme"

const Flag = ({
  children,
  className,
  img,
  reverse,
  top,
  bottom,
  responsive,
  flush,
  large,
  small,
  tiny,
  ...otherProps
}) => (
  <FlagObject
    className={className}
    reverse={reverse}
    top={top}
    bottom={bottom}
    responsive={responsive}
    flush={flush}
    large={large}
    small={small}
    tiny={tiny}
    {...otherProps}
  >
    <FlagImg>{img}</FlagImg>
    <FlagBody>{children}</FlagBody>
  </FlagObject>
)

const FlagImg = styled.div`
  display: table-cell;
  width: 1px;
  vertical-align: middle;
  padding-right: ${spacing.medium()};
  > img {
    max-width: none;
  }
  > svg {
    display: block;
  }
`

const FlagBody = styled.div`
  display: table-cell;
  vertical-align: middle;
  width: auto;

  &,
  > :last-child {
    margin-bottom: 0;
  }
`

const reverse = css`
  direction: rtl;
  > ${FlagImg}, > ${FlagBody} {
    direction: ltr;
  }

  > ${FlagImg} {
    padding-right: 0;
    padding-left: ${spacing.medium()};
  }
`

const reverseLarge = css`
  > ${FlagImg} {
    padding-right: 0;
    padding-left: ${spacing.large()};
  }
`

const reverseSmall = css`
  > ${FlagImg} {
    padding-right: 0;
    padding-left: ${spacing.small()};
  }
`

const reverseTiny = css`
  > ${FlagImg} {
    padding-right: 0;
    padding-left: ${spacing.tiny()};
  }
`

const top = css`
  > ${FlagImg}, > ${FlagBody} {
    vertical-align: top;
  }
`

const bottom = css`
  > ${FlagImg}, > ${FlagBody} {
    vertical-align: bottom;
  }
`

const flush = css`
  > ${FlagImg} {
    padding-right: 0;
    padding-left: 0;
  }
`

const large = css`
  > ${FlagImg} {
    padding-right: ${spacing.large()};
  }
  ${props => (props.reverse ? reverseLarge : null)};
`

const small = css`
  > ${FlagImg} {
    padding-right: ${spacing.small()};
  }
  ${props => (props.reverse ? reverseSmall : null)};
`

const tiny = css`
  > ${FlagImg} {
    padding-right: ${spacing.tiny()};
  }
  ${props => (props.reverse ? reverseTiny : null)};
`

const responsive = css`
  @media screen and (max-width: ${props => props.responsive}px) {
    &,
    > ${FlagImg}, > ${FlagBody} {
      display: block;
      direction: initial;
      padding: 0;
    }
    > ${FlagImg} {
      width: auto;
      padding-bottom: ${spacing.medium()};
    }

    ${FlagImg} > img {
      max-width: 100%;
    }
  }
`

export const FLAG_OBJECT = "FLAG_OBJECT"
const FlagObject = styled.div`
  display: table;
  width: 100%;
  border-spacing: 0;
  ${props => (props.reverse ? reverse : null)};
  ${props => (props.top ? top : null)};
  ${props => (props.bottom ? bottom : null)};
  ${props => (props.flush ? flush : null)};
  ${props => (props.large ? large : null)};
  ${props => (props.small ? small : null)};
  ${props => (props.tiny ? tiny : null)};
  ${props => (props.responsive > 0 ? responsive : null)};
  ${themify(FLAG_OBJECT)};
`

Flag.defaultProps = {
  className: "",
  reverse: null,
  top: null,
  bottom: null,
  responsive: 0,
  flush: null,
  large: null,
  small: null,
  tiny: null
}

Flag.propTypes = {
  children: themeProps.children.isRequired,
  img: themeProps.children.isRequired,
  className: PropTypes.string,
  reverse: PropTypes.bool,
  top: PropTypes.bool,
  bottom: PropTypes.bool,
  responsive: PropTypes.number,
  flush: PropTypes.bool,
  large: PropTypes.bool,
  small: PropTypes.bool,
  tiny: PropTypes.bool
}

export default Flag
