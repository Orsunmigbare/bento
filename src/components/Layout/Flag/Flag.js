import React from "react"
import PropTypes from "prop-types"
import styled, { css } from "styled-components"
import {
  commonPropTypes,
  themePropTypes
} from "../../../constants/themeContants"
import { applyVariants, spacing } from "../../../theming"

/**
 * The flag object is a design pattern similar to the media object, however it
 * utilises `display: table[-cell];` to give us control over the vertical
 * alignments of the text and image.
 */
const Flag = ({
  children,
  className,
  img,
  reverse,
  top,
  bottom,
  responsive,
  flush,
  large,
  small,
  tiny,
  ...otherProps
}) => (
  <FlagObject
    className={className}
    reverse={reverse}
    top={top}
    bottom={bottom}
    responsive={responsive}
    flush={flush}
    large={large}
    small={small}
    tiny={tiny}
    {...otherProps}
  >
    <FlagImg>{img}</FlagImg>
    <FlagBody>{children}</FlagBody>
  </FlagObject>
)

Flag.themeProps = {
  object: {
    name: "FLAG_OBJECT",
    description: "",
    type: themePropTypes.style
  }
}

const FlagImg = styled.div`
  margin-right: ${spacing.medium};
  > svg {
    display: block;
  }
`

const FlagBody = styled.div`
  flex: 1 1 0%;

  &,
  > :last-child {
    margin-bottom: 0;
  }
`

const reverse = css`
  > ${FlagImg} {
    order: 1;
    margin-right: 0;
    margin-left: ${spacing.medium};
  }
`

const reverseLarge = css`
  > ${FlagImg} {
    margin-right: 0;
    margin-left: ${spacing.large};
  }
`

const reverseSmall = css`
  > ${FlagImg} {
    margin-right: 0;
    margin-left: ${spacing.small};
  }
`

const reverseTiny = css`
  > ${FlagImg} {
    margin-right: 0;
    margin-left: ${spacing.tiny};
  }
`

const top = css`
  align-items: flex-start;
`

const bottom = css`
  align-items: flex-end;
`

const flush = css`
  > ${FlagImg} {
    padding-right: 0;
    padding-left: 0;
  }
`

const large = css`
  > ${FlagImg} {
    margin-right: ${spacing.large};
  }
  ${props => (props.reverse ? reverseLarge : null)};
`

const small = css`
  > ${FlagImg} {
    margin-right: ${spacing.small};
  }
  ${props => (props.reverse ? reverseSmall : null)};
`

const tiny = css`
  > ${FlagImg} {
    margin-right: ${spacing.tiny};
  }
  ${props => (props.reverse ? reverseTiny : null)};
`

const responsive = css`
  @media screen and (max-width: ${props => props.responsive}px) {
    display: block;

    > ${FlagImg}, > ${FlagBody} {
      display: block;
      margin: 0;
    }
    > ${FlagImg} {
      width: auto;
      margin-bottom: ${spacing.medium};
    }

    ${FlagImg} > img {
      max-width: 100%;
    }
  }
`

const FlagObject = styled.div`
  display: flex;
  align-items: center;
  ${props => (props.reverse ? reverse : null)};
  ${props => (props.top ? top : null)};
  ${props => (props.bottom ? bottom : null)};
  ${props => (props.flush ? flush : null)};
  ${props => (props.large ? large : null)};
  ${props => (props.small ? small : null)};
  ${props => (props.tiny ? tiny : null)};
  ${props => (props.responsive > 0 ? responsive : null)};
  ${applyVariants(Flag.themeProps.object)};
`

Flag.defaultProps = {
  className: "",
  reverse: null,
  top: null,
  bottom: null,
  responsive: 0,
  flush: null,
  large: null,
  small: null,
  tiny: null
}

Flag.propTypes = {
  children: commonPropTypes.children.isRequired,
  img: commonPropTypes.children.isRequired,
  className: PropTypes.string,
  reverse: PropTypes.bool,
  top: PropTypes.bool,
  bottom: PropTypes.bool,
  responsive: PropTypes.number,
  flush: PropTypes.bool,
  large: PropTypes.bool,
  small: PropTypes.bool,
  tiny: PropTypes.bool
}

export default Flag
