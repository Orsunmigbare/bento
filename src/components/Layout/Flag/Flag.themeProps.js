import { themePropTypes } from "../../../theming"

const themeProps = {
  object: {
    name: "FLAG_OBJECT",
    description: "",
    type: themePropTypes.style
  }
}

export default themeProps
