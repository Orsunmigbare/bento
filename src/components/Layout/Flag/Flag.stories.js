import React from "react"
import { storiesOf } from "@storybook/react"
import Flag from "./Flag"
import docs from "./Flag.md"

storiesOf("components|base/Layout/Flag", module)
  .add("No props", () => <Flag />, { info: { text: docs } })
  .add(
    "With img",
    () => (
      <Flag
        responsive={800}
        img={
          <img
            src="https://www.w3schools.com/w3css/img_snowtops.jpg"
            width="100"
            height="100"
          />
        }
      />
    ),
    { info: { text: docs } }
  )
