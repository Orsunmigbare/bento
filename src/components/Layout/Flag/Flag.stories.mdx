import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks"
import Flag from "./Flag"

<Meta title="Components|Layout/Flag" component={Flag} />

# Flag
This is the Flag component. The component can be used to display a list with images.

```javascript
import { Flag } from "@staccx/bento"
```

## Types of Flags
There are several ways to display the Flag component.

## Standard Flag

<Preview>
  <Story name="Standard">
    <Flag
    responsive={800}
    img={<img src="http://via.placeholder.com/150x150" widt="100" height="100" />}>
    Text for flag-object, the standard way.
  </Flag>
  </Story>
</Preview>

## Reversed Flag

<Preview>
  <Story name="Reversed">
    <Flag
    reverse
    responsive={800}
    img={<img src="http://via.placeholder.com/150x150" widt="100" height="100" />}>
    Text for flag-object when the image is reversed
  </Flag>
  </Story>
</Preview>

## Positioning the text
There are three ways to position the text: top, bottom and flush.

<Preview>
  <Story name="top">
    <Flag
    top
    responsive={800}
    img={<img src="http://via.placeholder.com/150x150" widt="100" height="100" />}>
    Text for flag-object positioned to the top
  </Flag>
  </Story>
</Preview>

# Props for Flag

<Props of={Flag} />
