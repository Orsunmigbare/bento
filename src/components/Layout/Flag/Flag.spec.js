import React from "react"
import renderer from "react-test-renderer"
import { ThemeProvider } from "styled-components"
import baseTheme from "../../../theming/themes/baseTheme"
import Flag from "./Flag"

const Child = () => <p>Body</p>

describe("Flag", () => {
  describe("Snapshots", () => {
    it("No props", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag>
              <Child />
            </Flag>
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("img", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag
              img={
                <img
                  src="http://via.placeholder.com/150x150"
                  width="100"
                  height="100"
                />
              }
            />
            <Child />
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("className", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag className="className" />
            <Child />
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("reverse", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag reverse>
              <Child />
            </Flag>
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("top", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag top>
              <Child />
            </Flag>
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("bottom", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag bottom>
              <Child />
            </Flag>
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("responsive", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag responsive={10}>
              <Child />
            </Flag>
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("flush", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag flush>
              <Child />
            </Flag>
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("large", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag large>
              <Child />
            </Flag>
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("small", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag small>
              <Child />
            </Flag>
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("tiny", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag tiny>
              <Child />
            </Flag>
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
    it("All", () => {
      const tree = renderer
        .create(
          <ThemeProvider theme={baseTheme}>
            <Flag
              tiny
              responsive={10}
              bottom
              className="className"
              img={
                <img
                  src="http://via.placeholder.com/150x150"
                  width="100"
                  height="100"
                />
              }
            >
              <Child />
            </Flag>
          </ThemeProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
