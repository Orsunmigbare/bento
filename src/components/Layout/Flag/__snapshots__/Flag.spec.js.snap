// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flag Snapshots All 1`] = `
<div
  className="sc-AxiKw kSCNI className"
>
  <div
    className="sc-AxjAm hBateQ"
  >
    <img
      height="100"
      src="http://via.placeholder.com/150x150"
      width="100"
    />
  </div>
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots No props 1`] = `
<div
  className="sc-AxiKw jqePkp"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots bottom 1`] = `
<div
  className="sc-AxiKw cqpOYn"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots className 1`] = `
Array [
  <div
    className="sc-AxiKw jqePkp className"
  >
    <div
      className="sc-AxjAm hBateQ"
    />
    <div
      className="sc-AxirZ gunZf"
    />
  </div>,
  <p>
    Body
  </p>,
]
`;

exports[`Flag Snapshots flush 1`] = `
<div
  className="sc-AxiKw pIBQk"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots img 1`] = `
Array [
  <div
    className="sc-AxiKw jqePkp"
  >
    <div
      className="sc-AxjAm hBateQ"
    >
      <img
        height="100"
        src="http://via.placeholder.com/150x150"
        width="100"
      />
    </div>
    <div
      className="sc-AxirZ gunZf"
    />
  </div>,
  <p>
    Body
  </p>,
]
`;

exports[`Flag Snapshots large 1`] = `
<div
  className="sc-AxiKw bHnqFO"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots large reverse 1`] = `
<div
  className="sc-AxiKw iMqEqQ"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots responsive 1`] = `
<div
  className="sc-AxiKw eDcKhj"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots reverse 1`] = `
<div
  className="sc-AxiKw hIQzje"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots small 1`] = `
<div
  className="sc-AxiKw fkuugJ"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots small reverse 1`] = `
<div
  className="sc-AxiKw dmOrvw"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots tiny 1`] = `
<div
  className="sc-AxiKw gGpGSE"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots tiny reverse 1`] = `
<div
  className="sc-AxiKw irORNo"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;

exports[`Flag Snapshots top 1`] = `
<div
  className="sc-AxiKw gmGek"
>
  <div
    className="sc-AxjAm hBateQ"
  />
  <div
    className="sc-AxirZ gunZf"
  >
    <p>
      Body
    </p>
  </div>
</div>
`;
